### PR TITLE
2426-unable-to-apply-max-width-tokens-on-component-instances

### PR DESCRIPTION
**Pull request by robinhoodie0823**
### Why does this PR exist?

Closes #0000 

### Additional Notes (if any)

https://github.com/tokens-studio/figma-plugin/assets/103296157/1c6197a3-829a-4705-80f0-2897dc2e8c86

### CODE REVIEW

File: packages/tokens-studio-for-figma/src/plugin/setValuesOnNode.ts [hunk 1]
```diff
@@ -47,7 +47,6 @@ export default async function setValuesOnNode(
   baseFontSize = defaultBaseFontSize,
 ) {
   const stylePathSlice = ignoreFirstPartForStyles ? 1 : 0;
```
Comment (six7) [inline packages/tokens-studio-for-figma/src/plugin/setValuesOnNode.ts:49]:
```suggestion
  const stylePathSlice = ignoreFirstPartForStyles ? 1 : 0;

```

File: packages/tokens-studio-for-figma/src/app/store/updateSources.tsx [hunk 2]
```diff
@@ -143,7 +143,6 @@ export default async function updateTokensOnSources({
       dispatch,
     });
   }
```
Comment (six7) [inline packages/tokens-studio-for-figma/src/app/store/updateSources.tsx:145]:
```suggestion
  }

```

**Review by six7 — APPROVED** [review]

**Comment by six7** [comment]
Will be fixed in 2.0

**Comment by SamIam4Hyma** [comment]
Added to Featurebase Roadmap-	[🐛 V2 Bug Fixes](https://tokensstudio.featurebase.app/p/v2-bug-fixes)

**Comment by SamIam4Hyma** [comment]
Added to Featurebase Roadmap-	[🐛 V2 Bug Fixes](https://tokensstudio.featurebase.app/p/v2-bug-fixes)

**Pull request merged**